styles.css

```diff
@@ -0,0 +1,958 @@
+}
+
+.deals a {
+  color: #000;
+  font-size: 12px;
+}
+
+.img-des a {
+  color: white;
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: bolder;
+}
+
+.funding_img a {
+  color: black;
+  font-size: 14px;
+  font-weight: 400px;
+}
+
+.nav-left li a,
+.nav-right li a {
+  color: rgb(197, 196, 196);
+}
+
+.second-list li a,
+#logo-container a span {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  color: #aaa;
+}
+
+.news_content a:hover {
+  color: gray;
+}
+
+.nav-list li {
+  display: inline-block;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  font-size: 14px;
+  list-style: none;
+  padding-left: 20px;
+}
+
+ul li ul.drop1 {
+  left: 8%;
+  padding-left: 5px;
+  width: 15%;
+}
+
+ul li ul.drop2 {
+  left: 15%;
+  padding-left: 5px;
+}
+
+.secondary_footer ul {
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+ul li:hover > ul,
+ul li:focus-within > ul,
+ul li ul:hover {
+  display: block;
+  opacity: 1;
+  visibility: visible;
+}
+
+.links li {
+  background-color: #333;
+  border-radius: 50%;
+  border: 2px solid #333;
+  color: white;
+  cursor: pointer;
+  font-size: 14px;
+  padding: 10px;
+  margin: 5px;
+}
+
+.caption a:hover {
+  color: #969595;
+}
+
+.nav-right li {
+  display: inline-block;
+  text-align: center;
+}
+
+.footer-links li {
+  padding: 8px;
+}
+
+.secondary_footer li {
+  font-size: 13px;
+}
+
+ul li ul li:first-child {
+  border-top: none;
+}
+
+.nav-left {
+  margin: 0;
+  padding: 0;
+  height: 33px;
+}
+
+.grid-nav-wrapper {
+  background: #131313;
+  display: flex;
+  flex-direction: row;
+  height: 31px;
+  justify-content: space-between;
+  padding: 0.45em 10px 0 10px;
+}
+
+.nav-right {
+  margin: 0;
+}
+
+.latest-links a:hover {
+  color: #000;
+}
+
+.deals a:hover {
+  color: gray;
+}
+
+.img-des a:hover {
+  color: gray;
+}
+
+.nav-left li a:hover,
+.nav-right li a:hover {
+  color: #fff;
+}
+
+.second-list {
+  background: #fff;
+  margin: 0;
+  padding: 20px 15px;
+  height: 55px;
+  width: 100%;
+}
+
+.nav-icons {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+#search-icon {
+  background: #fff;
+  border-left: 1px solid #eee;
+  padding: 19px 20px;
+}
+
+.fa-search {
+  color: #aaa;
+}
+
+#menu-icon {
+  border-left: 1px solid #eee;
+  display: none;
+  padding: 20px;
+}
+
+.fa-bars {
+  color: #f42;
+}
+
+/* section one : */
+
+.first_content {
+  width: 100%;
+  margin-top: 20px;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-column-gap: 1%;
+}
+
+.first_article {
+  margin-left: 1%;
+  background-image: linear-gradient(to top left, rgb(184, 3, 3), rgb(87, 2, 2), rgb(8, 5, 5), transparent), url(images/img6.jpg);
+  opacity: 0.8;
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: relative;
+  height: 460px;
```

Comment:
**[REQUIRED]** All the articles should have the same height. The first article should not contain logo in smaller screens. All articles should look similar in the smaller screens. Give them a dark background gradient to make the white text more visible